contrib/download-frozen-image-v2.sh

```diff
@@ -201,6 +201,58 @@ handle_single_manifest_v2() {
 	manifestJsonEntries=("${manifestJsonEntries[@]}" "$manifestJsonEntry")
 }
 
+get_target_arch() {
+	if [ -n "${TARGETARCH}" ]; then
+		echo "${TARGETARCH}"
+		return 0
+	fi
+
+	if type go > /dev/null; then
+		echo "$(go env GOARCH)"
+		return 0
+	fi
+
+	if type dpkg > /dev/null; then
+		debArch="$(dpkg --print-architecture)"
+		case "${debArch}" in
+			armel | armhf)
+				echo "arm"
+				return 0
+				;;
+			*)
+				echo "${debArch}"
+				return 0
+				;;
+		esac
+	fi
```

Comment:
This isn't exactly right -- the "little endian" architectures are dyslexic between Go and Debian (`mips64le` vs `mips64el`, etc).

I guess we could do something like this:

```suggestion
		case "${debArch}" in
			armel | armhf)
				echo "arm"
				return 0
				;;
			*64el)
				echo "${debArch%el}le"
				;;
			*)
				echo "${debArch}"
				return 0
				;;
		esac
	fi
```